lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/google.rb

```diff
@@ -366,6 +365,11 @@ def create_disk(name, size_gb, zone_name, source_image)
         fail "Disk #{disk.name} already exists with different specifications"
       end
     end
+    if source_image.nil?
```

Comment:
Fair enough. I'm leaving name explicit, though. It'll be easier to find in the logs, IMO.